sycl/plugins/unified_runtime/ur/adapters/level_zero/event.cpp

```diff
@@ -935,7 +935,7 @@ ur_result_t CleanupCompletedEvent(ur_event_handle_t Event, bool QueueLocked,
     }
     if (DepEventKernel) {
       ReleaseIndirectMem(DepEventKernel);
-      // UR_CALL(piKernelRelease(DepEventKernel));
+      UR_CALL(urKernelRelease(DepEventKernel));
```

Comment:
do you know why this was commented out? wasn't this causing massive memory leaks?

Reply:
@smaslov-intel : when I was doing the porting, I was commenting out some lines that were causing errors just to move more rapidly, but before completing the porting, I put them back. I missed doing that for this line, but it was there originally in the PI code:

https://github.com/intel/llvm/commit/3c35b1b59093a2b34dcaf803f995b3e3ab5c0cde

![image](https://github.com/intel/llvm/assets/8868111/2113d0e9-f382-49a9-92ed-f05df9c70da4)

Reply:
Right, and I think it is indeed needed. Just wonder why we didn't end up finding it before since it is expected to cause massive leaks